src/commands/convert/conveyor2hdf.ts

```diff
@@ -0,0 +1,36 @@
+import {Command, Flags} from '@oclif/core'
```

Comment:
https://github.com/mitre/saf/blob/main/src/commands/convert/index.ts
hasn't been added to the generic command

Reply:
added to index.ts